dist/tools/kconfiglib/examples/select_specific_implementation_from_module/generate_example.py

```diff
@@ -0,0 +1,56 @@
+import argparse
```

Comment:
Why do we need a script to generate the example?

Reply:
Take care of some of the boilerplate stuff, this can also show what is convention and what is specific to the module.